postprocessing/science/compute_isochrones.py

```diff
@@ -0,0 +1,250 @@
+# Script to compute isochrones (for S-wave radiation) of a single receiver from SeisSol's surface/fault.xdmf output
+# Isochrones are calculated from RT output and peak slip rate times
+
+import numpy as np
+import seissolxdmf
+import argparse
+import seissolxdmfwriter as sxw
+import timeit
+from multiprocessing import Pool, cpu_count
+
+parser = argparse.ArgumentParser(description='get isochrones from SeisSol\'s xdmf output')
+
+parser.add_argument('filename', help='path+filename-surface.xdmf')
+
+parser.add_argument("x", nargs=1, type=float, help="receiver x")
+parser.add_argument("y", nargs=1, type=float, help="receiver y")
+parser.add_argument("--z", nargs=1, type=float, default=([0.]), help="receiver z")
+
+parser.add_argument('--faultXdmf', type=str, metavar=('-fault.xdmf'), 
+                    help='provide path+filename-fault.xdmf; only needed when the path differs from the -surface.xdmf file')
+
+parser.add_argument('--output', choices=["numpy","xdmf","both"], default="xdmf", 
+                    help='choose the output format')
+
+parser.add_argument('--sliprateThreshold', nargs=1, default=([0.05]), metavar=('peak sliprate threshold in m/s'),
+                    help='fault cells below the treshold are assumed to not radiate waves' , type=float)
+
+parser.add_argument('--sliprateParameters', choices=["absolute","components","both"], default="both", 
+                    help="absolute: compute isochrones from peak times of absolute slip rates;"+ 
+                    " components: peak slip rate times are computed for SRs and SRd separately")
+
+parser.add_argument('--medianFilterRadius', nargs=1, default=([0.]), metavar=('radius of the median filter in m'),
```

Comment:
maybe remove the default, and add required=True, forcing the user to have this (maybe) important decision?

Reply:
yes, good idea